packages/calcite-components/src/components/list/list.stories.ts

```diff
@@ -1137,3 +1150,23 @@ export const dragEnabledNestedListsIndirectChildren = (): string =>
     <div><calcite-list-item label="Depth 1" description="Item 6"></calcite-list-item></div>
     <div><calcite-list-item drag-disabled label="Depth 1" description="Item 7"></calcite-list-item></div>
   </calcite-list>`;
+
+export const nonInteractive = (): string => html`
```

Comment:
Can you merge these stories into one with a descriptor for each use case (e.g., [slider fill placements](https://github.com/Esri/calcite-design-system/blob/4c2c623d59441be8ddc66866748b7b104810b5ac/packages/calcite-components/src/components/slider/slider.stories.ts#L466-L524))?